Add LIBPTHREAD_SO macro to gnu/lib-names.h and disable nscd build

No topgit branch, TODO?

---
 nscd/Makefile         |    2 +-
 scripts/lib-names.awk |    2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

--- a/scripts/lib-names.awk
+++ b/scripts/lib-names.awk
@@ -61,6 +61,8 @@
     cmd = "LC_ALL=C sort"; print lines["DEFAULT"] | cmd; close(cmd);
   }
 
+  print "#define LIBPTHREAD_SO \"libpthread.so.0.3\""
+
   print "";
   print "#endif	/* gnu/lib-names.h */"
 }
--- a/nscd/Makefile
+++ b/nscd/Makefile
@@ -43,7 +43,7 @@
 # own copy of nis_hash.
 nscd-modules += nis_hash
 endif
-ifeq ($(have-thread-library),yes)
+ifeq ($(have-thread-library),yesreallyyes)
 
 ifneq (yesyes,$(have-fpie)$(build-shared))
 others-$(OPTION_EGLIBC_INET) += nscd
